bin/improver-nbhood-iterate-with-mask

```diff
@@ -0,0 +1,131 @@
+    parser.add_argument('input_mask_filepath', metavar='INPUT_MASK_FILE',
+                        help='A path to an input mask NetCDF file to be '
+                             'used to mask the input file.')
+    parser.add_argument('output_filepath', metavar='OUTPUT_FILE',
+                        help='The output path for the processed NetCDF.')
+    group = parser.add_mutually_exclusive_group()
+    group.add_argument('--radius', metavar='RADIUS', type=float,
+                       help='The radius (in m) for neighbourhood processing.')
+    group.add_argument('--radii-by-lead-time',
+                       metavar=('RADII_BY_LEAD_TIME', 'LEAD_TIME_IN_HOURS'),
+                       nargs=2,
+                       help='The radii for neighbourhood processing '
+                       'and the associated lead times at which the radii are '
+                       'valid. The radii are in metres whilst the lead time '
+                       'has units of hours. The radii and lead times are '
+                       'expected as individual comma-separated lists with '
+                       'the list of radii given first followed by a list of '
+                       'lead times to indicate at what lead time each radii '
+                       'should be used. For example: 10000,12000,14000 1,2,3 '
+                       'where a lead time of 1 hour uses a radius of 10000m, '
+                       'a lead time of 2 hours uses a radius of 12000m, etc.')
+    parser.add_argument('--ens_factor', metavar='ENS_FACTOR', type=float,
+                        default=1.0,
+                        help='The factor with which to adjust the '
+                        'neighbourhood size for more than one '
+                        'ensemble member. If ens_factor = 1.0 this '
+                        'essentially conserves ensemble members if '
+                        'every grid square is considered to be the '
+                        'equivalent of an ensemble member.'
+                        'Optional, defaults to 1.0.')
+    parser.add_argument('--sum_or_fraction', default="fraction",
+                        choices=["sum", "fraction"],
+                        help='The neighbourhood output can either be in the '
+                             'form of a sum of the neighbourhood, or a '
+                             'fraction calculated by dividing the sum of the '
+                             'neighbourhood by the neighbourhood area. '
+                             '"fraction" is the default option.')
+    parser.add_argument('--re_mask', action='store_true',
+                        help='If re_mask is set (i.e. True), the output cube '
+                             'following neighbourhood processing is '
+                             're-masked. This re-masking removes any values '
+                             'that have been generated by neighbourhood '
+                             'processing at grid points within the masked '
```

Comment:
can we change this to: This re-masking removes any values '
 +                             'that have been generated by neighbourhood '
 +                             'processing at points that were originally masked.